main.py

```diff
@@ -43,6 +44,15 @@ async def runes(ctx, champ_name=None, role=None, num=None):
 async def builds(ctx, champ_name=None, role=None):
     await ctx.send(builds_command(champ_name, role))
 
+@bot.command()
+async def top(ctx, sum_name=None, ext=None):
+    await ctx.send(topchampions_command(ctx, sum_name, ext))
+
+
+@bot.command()
+async def top(ctx, sum_name=None, ext=None):
```

Comment:
I think the name `topchamps` is better than just `top`. Up for debate.

Also try changing `ext` to `queue`. Ext is used for extra information, in this case, queue is needed and is not considered extra.